cmd/inch/main.go

```diff
@@ -86,7 +86,11 @@ func (m *Main) ParseFlags(args []string) error {
 	}
 
 	// Parse tag cardinalities.
-	for _, s := range strings.Split(*tags, ",") {
+	for i, s := range strings.Split(*tags, ",") {
+		if i == 0 {
+			// We're overriding default tag cardinalities
+			m.inch.Tags = []int{}
+		}
```

Comment:
Won't `strings.Split()` always return a slice of at least one? This will always execute.

Reply:
Yes it will, but `*tags` is guaranteed to be either `10,10,10` or some other non-empty value, e.g., `-t 100`. Either we'll re-add the default `Simulator` tags, or we'll add the custom tags defined by the `-t` flag.

This just seemed cleaner than inspecting the result of the `strings.Split` and figuring out what to do.